.github/workflows/cli.yml

```diff
@@ -40,22 +40,40 @@ jobs:
         ../packages/cli/dist/bin/index.js dataset --help
         ../packages/cli/dist/bin/index.js bip --help
         ../packages/cli/dist/bin/index.js serve --help
-    - name: test client basic operate init
+  basic-usage:
+    name: Test CLI basic usage on node ${{ matrix.node_version }} and ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        node_version: ['12']
+        os: [ubuntu-latest, macOS-latest]
+    steps:
+    - uses: actions/checkout@v1
+    - name: Use Node.js ${{ matrix.node_version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node_version }}
+    - name: building
+      run: |
+        npm install
+        npm run build
+    - name: create workspace
+      run: mkdir -p myworkspace
+    - name: test client basic usage init
       working-directory: ./myworkspace
       run: |
+        echo "before init:"
+        pwd
+        ls
         mkdir init1
         cd init1
         ../../packages/cli/dist/bin/index.js init
+        ../../packages/cli/dist/bin/index.js run ./examples/pipelines/text-bayes-classification.json
```

Comment:
It's good to test the "run" in another step, which makes people to find the run result easier.